spec/models/refinery/jobs/job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+module Refinery
+  module Jobs
+    describe Job, type: :model do
+      let(:job) {FactoryGirl.create(:job)}
+      describe 'validations' do
+        it 'requires title and description' do
+          #Negative tests
+          expect(FactoryGirl.build(:job, description: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job, title: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job, title: '')).not_to be_valid
+          expect(FactoryGirl.build(:job, description: '')).not_to be_valid
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.